storage/netstore.go

```diff
@@ -201,7 +205,8 @@ func (n *NetStore) Get(ctx context.Context, mode chunk.ModeGet, req *Request) (c
 			if ok {
 				ch, err = n.RemoteFetch(ctx, req, fi)
 				if err != nil {
-					if n.recoveryCallback != nil {
+					if n.recoveryCallback != nil && publisher != "" {
+						log.Debug("gp netstore recovery triggered")
 						n.recoveryCallback(ctx, ref)
 						time.Sleep(500 * time.Millisecond) // TODO: view what the ideal timeout is
```

Comment:
did you try measuring how long this actually takes?

i think we should either remove this retry, or use a `Sleep` amount which could realistically work (`500` is too little for even 3 nodes).

we can also look into outputting a "recovery triggered, retry later" message into the CLI, although this could be done in a different issue

Reply:
relvant: #2172

Reply:
It takes around 2-3 seconds which is more than this Sleep.
The problem is that a 3-second delay will halt the execution and also it can differ on other setups.

I would prefer to remove this and emit a message with a status, so the client can retry at their own retry interval.
What do you think @zelig?

Reply:
in the book i resurrected error code 420  ('enhance your calm' of twitter fame) to be returned if recovery was initiated. Indeed the client could then just retry

Reply:
> in the book i resurrected error code 420 ('enhance your calm' of twitter fame) to be returned if recovery was initiated. Indeed the client could then just retry

love this idea